cfg/config.go

```diff
@@ -200,6 +200,8 @@ type MetadataCacheConfig struct {
 
 	TtlSecs int64 `yaml:"ttl-secs"`
 
+	NegativeTtlSecs int64 `yaml:"negative-ttl-secs"`
```

Comment:
You missed initializing flag in [params.yaml](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/cfg/params.yaml)

Reply:
added